### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by udev-21**
Added two sections: "About me" and  "Works"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,9 +41,127 @@ <h2 class="green">I’m a software developer</h2>
         <div class="next-section">
             <img class="next-section-svg" src="assets/icons/round-arrow-down.svg" alt="">
         </div>
+        <div class="background-image-wrapper">
+            <img id="into-background" src="assets/images/background-mobile.svg" alt="">
```
Comment (emmiiorji) [inline index.html:45]: {Optional] I kindly suggest that we modify our intro section to match the Figma design we've chosen. Not only does it look better, but you'll also get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible. Therefore, sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now.

Our design currently:
image


Figma design:
image

File: index.html [hunk 2]
```diff
@@ -41,9 +41,127 @@ <h2 class="green">I’m a software developer</h2>
...
+                    </div>
+                    <div class="project-stacks">
+                        <div>Ruby on Rails</div>
+                        <div>css</div>
+                        <div>JavaScript</div>
+                        <div>html</div>
+                    </div>
+                    <div class="button-wrapper">
+                        <a  href="#" class="button">See Project</a> 
+                    </div>
```
Comment (emmiiorji) [inline index.html:82]: Could we implement the button interaction for our chosen button color scheme as given by the design?

Here are the interactions given:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @udev-21, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Almost there


## Good Highlights ✅

- Linter checks are passing
- Good commit messages
- Adherence to GitHub flow


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider the following:
- I suggest that your remove the foot note on your README file (talking about using MIT license). It is only a note to the author of the project and removing would make the file more concise.
image

- The font size of our intro paragraph seems smaller than the that given by the design. It will be great if we could make them match

Our design currently:
image

Figma design:
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (udev-21) [reply index.html:45]: I did the font size exactly as in figma.
I think it depends on device's width.

File: index.html (hunk 2 above)
Comment (udev-21) [reply index.html:82]: got it, added hover and active interactions

**Review by mwanawabangona — APPROVED** [review]
Hi @udev-21 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (emmiiorji) [reply index.html:45]: Hi @udev-21,

Great.

However, the device width used to take the screenshot is same as the Figma design (375px)

**Pull request merged**